### ISSUE TITLE
Error on Entity Creation

### ISSUE THREAD

**Issue by campmlc** [ArctosDB/arctos#7839]
Error ID: 083EA180-7634-4650-AAC27243B71726A4
Trying to create an Entity using search on studbook number 510 for a single Mexican wolf with 2 MSB:Mamm and 2 DGR:Mamm catalog numbers. I get the following error: "can't cast empty string to a number value".

I have spent an hour trying to clean up these two DGR records in the larger context of #7808. In this context, it will not be possible to deal with all of these legacy issues and catalog numbers if we want to proceed with #7808. We need a way to move forward without getting tangled in all of these legacy problems. Otherwise, I need at least a solid month of my and  @dustymc  time just to deal with the MSB related issues.

**Comment by campmlc** [comment]
https://arctos.database.museum/search.cfm?customoidoper=LIST&oidtype=studbook&oidnum=510

**Comment by campmlc** [comment]
The two DGR:Mamm records are encumbered, but they have the legacy object tracking path and loan dependencies. I've cloned them as the MSB:Mamm records, and I've added MSB:Mamm parts to the legacy loan with a note that these were loaned as the DGR:Mamm catalog numbers. I've barcoded and added the barcode to one legacy DGR locator sample using the scary red box, after re-converting the barcode from cryovial to cryovial label.. For the other, not yet - it's in LN2. I'd like to create redirects for the original DGR:Mamm records before removing all the dependencies, correcting all the object tracking pathways, and deleting the records. This is far too convoluted and challenging and time consuming, and currently requires being done one at a time. After 2 hours, I'm only part of the way through. This needs to be done for any DGR guid that has loan and DGR Locator dependencies.
